codespell_lib/data/dictionary.txt

```diff
@@ -892,8 +892,11 @@ againt->against
 againts->against
 agaisnt->against
 agaist->against
+agancies->agencies
+agancy->agency
 aganda->agenda
 aganist->against
+agant->agent
```

Comment:
...and the plural? :wink: (I kinda automatically assumed you'd be adding that too!)
Although perhaps it should be `agants->agents, against, giants,` ? :man_shrugging:

Reply:
You should have suggested it then! :stuck_out_tongue: 

I've added against as a miss and a swap, but the a/i position and keyboard distance seems too great a stretch IMHO.